README.md

```diff
@@ -65,4 +82,4 @@ in your environment variables.
 
 Please see the LICENSE file for the included license information.
 
-Copyright 2013 by authors and contributors.
+Copyright 2017 by authors and contributors.
```

Comment:
I think technically it should be `2013-2017`, but I'm not sure it strictly matters.